vic/drivers/classic/src/vic_classic.c

```diff
@@ -223,15 +237,17 @@ main(int   argc,
                 /**************************************************
                    Compute cell physics for 1 timestep
                 **************************************************/
+                timer_start(&cell_timer);
```

Comment:
does it need to accumulate the time for each cell? or you just simply want to write out the time for each time step. it will be great to have a summary for each cell.

the approach is quite different than I used. I usually track the time of functions instead of cells.